packages/esm-tb-app/src/constants.ts

```diff
@@ -7,3 +7,11 @@ export const contactMethod = '166456AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
 export const ClientListingEncounterType = '70d2b2f6-860b-438c-994b-c28e863dca34';
 export const CaseId = '162576AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
 export const DateContacted = '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+
+//TB Follow-up
+export const VisitDate = '160753AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+export const TbFollowUpEncounterType = '91654d2c-d266-4661-9110-a239cf82d5b8';
+export const FollowUpCaseId = '162576AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+export const MonthOfTreatment = '1418AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+export const AdherenceAssessment = '164075AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
+export const NextAppointmentDate = '5096AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA';
```

Comment:
These need to move to the config

Reply:
@pirupius Please explain this?

Reply:
Take a look at this PR https://github.com/UCSF-IGHS/openmrs-esm-ohri/pull/1712

Reply:
Have that PR merged so we can have access to that file for the TB esm?